README.md

```diff
@@ -1824,6 +1825,36 @@ Create a new function that limits calls to _func_ to once every given timeframe.
 
 ## Lang
 
+### _.castArray
+
+Puts the value into an array of length one if it is not already an array.
+
+```js
+// Underscore/Lodash
```

Comment:
I can't find any references that underscore has this function.

Reply:
Good catch, I removed that and added the rules.